lib/rubocop/cop/util.rb

```diff
@@ -137,18 +137,19 @@ def range_with_surrounding_comma(range, side = :both)
         Parser::Source::Range.new(buffer, begin_pos, end_pos)
       end
 
-      def range_with_surrounding_space(range, side = :both, with_newline = true)
+      def range_with_surrounding_space(range,
+                                       side = :both,
```

Comment:
I guess that's a good opportunity to replace those optional args with keyword args.

Reply:
Maybe in another commit, though.